Angular/src/app/project-info/custom.datepipe.ts

```diff
@@ -7,6 +7,6 @@ import { DatePipe } from '@angular/common';
   export class CustomDatePipe extends
                DatePipe implements PipeTransform {
     transform(value: any, args?: any): any {
-      return super.transform(value, 'y MM d H:mm');
+      return super.transform(value, 'y/MM/dd H:mm');
```

Comment:
most probably we will have the same dates format across the project - pls move it to the /shared/pipes and 'y/MM/dd H:mm' into Constants->DateFormats->LongDateString